gui/src/main/resources/css/graph.css

```
@@ -4,10 +4,11 @@ graph{
 
 node {
     size: 64px;
-    fill-mode: image-scaled;
+    fill-mode: image-scaled-ratio-max;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)